versioned_docs/version-5.x/nesting-navigators.md

```diff
@@ -105,23 +109,30 @@ In your app, you will probably use these patterns depending on the behavior you
 
 Consider the following example:
 
+<samp id="nested-navigator-screen" />
+
 ```js
 function Root() {
   return (
-    <Stack.Navigator>
-      <Stack.Screen name="Profile" component={Profile} />
+    <Drawer.Navigator>
+      <Drawer.Screen name="Home" component={Home} />
+      <Drawer.Screen name="Profile" component={Profile} />
       <Stack.Screen name="Settings" component={Settings} />
-    </Stack.Navigator>
+    </Drawer.Navigator>
   );
 }
 
 function App() {
   return (
     <NavigationContainer>
-      <Drawer.Navigator>
-        <Drawer.Screen name="Home" component={Home} />
-        <Drawer.Screen name="Root" component={Root} />
-      </Drawer.Navigator>
+      <Stack.Navigator>
+        <Stack.Screen
+          name="Root"
+          component={Root}
+          options={{ headerShown: false }}
+        />
+        <Stack.Screen name="Feed" component={Feed} />
+      </Stack.Navigator>
     </NavigationContainer>
```

Comment:
To keep the text above consistent with the example please bring back the previous code as it was just for v5